INSTALLER-HACKING.md

```diff
@@ -0,0 +1,63 @@
+# Running local builds with the installer
+
+If you want to run a local build with the installer, you need to do a bit of hacking. There is a script that will do most of the dirty work for you.
+
+## Step 0: Prepare your credentials
+
+You will need to do this once, and every time your token resets.
+
+1. Get your CI credentials by logging on to the [CI server](https://api.ci.openshift.org/console) using your GitHub account and clicking _Copy Login Command_ from the menu on the top right.
+
+2. Execute that command, which should look like
+
+```
+oc login https://api.ci.openshift.org --token=XXXX
+```
+
+3. Copy that token, and use it to log in to the registry with podman:
+
+```
+podman login -u <YOUR GITHUB ID> -p <YOUR TOKEN> registry.svc.ci.openshift.org
+```
+
+Tokens expire regularly - about every week.
+
+
+## Step 1: Prepare the cluster & environment
+
+You will need to do this for every new cluster
```

Comment:
After I destroy the cluster, can I run again from the same `$CLUSTER_DIR`? If so, which parts of this can I skip the second time around?